src/scripts/main.js

```diff
@@ -1,3 +1,317 @@
 'use strict';
 
-// write your code here
+require('regenerator-runtime/runtime');
+
+const startButton = document.querySelector('button.button.start');
+const listCells = document.querySelectorAll('.innerField');
+const scoreField = document.querySelector('span.game-score');
+let firstField = 0;
+let secondField = 0;
+let isMovingNow = false;
+
+const winMessage = document.querySelector('.message-win');
+const loseMessage = document.querySelector('.message-lose');
+
+const findRandNunb = (min, max) => {
+  return Math.floor(Math.random() * (max - min)) + min;
+};
+
+startButton.addEventListener('click', () => {
+  startGame();
+});
+
+function randomNumbForStart() {
+  const min = 1;
+  const max = 16;
+
+  firstField = findRandNunb(min, max);
+  secondField = findRandNunb(min, max);
+
+  while (secondField === firstField) {
+    secondField = findRandNunb(min, max);
+  }
+
+  return [firstField, secondField];
+}
+
+function startGame() {
+  const arr = randomNumbForStart();
```

Comment:
fix it please
https://github.com/mate-academy/js_2048_game/pull/196/files#r1101352371